<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2018-09-01T19:05:14Z" name="Suite" finished-at="2018-09-01T19:05:37Z" duration-ms="22623">
    <groups>
    </groups>
    <test started-at="2018-09-01T19:05:14Z" name="Test" finished-at="2018-09-01T19:05:37Z" duration-ms="22623">
      <class name="testNGListeners1.TestCase2">
        <test-method signature="GoogleTitleMatch()[pri:0, instance:testNGListeners1.TestCase2@4229bb3f]" started-at="2018-09-01T19:05:33Z" name="GoogleTitleMatch" finished-at="2018-09-01T19:05:33Z" duration-ms="6" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at testNGListeners1.TestCase2.GoogleTitleMatch(TestCase2.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GoogleTitleMatch -->
        <test-method signature="GoogleTitleVerify()[pri:0, instance:testNGListeners1.TestCase2@4229bb3f]" started-at="2018-09-01T19:05:33Z" name="GoogleTitleVerify" finished-at="2018-09-01T19:05:34Z" duration-ms="937" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GoogleTitleVerify -->
      </class> <!-- testNGListeners1.TestCase2 -->
      <class name="testNGListeners1.TestCase1">
        <test-method is-config="true" signature="InvokeBrowser()[pri:0, instance:testNGListeners1.TestCase1@7fee8714]" started-at="2018-09-01T19:05:14Z" name="InvokeBrowser" finished-at="2018-09-01T19:05:24Z" duration-ms="9965" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InvokeBrowser -->
        <test-method is-config="true" signature="InvokeBrowser()[pri:0, instance:testNGListeners1.TestCase2@4229bb3f]" started-at="2018-09-01T19:05:24Z" name="InvokeBrowser" finished-at="2018-09-01T19:05:31Z" duration-ms="6931" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InvokeBrowser -->
        <test-method signature="AmazonTitleMatch()[pri:0, instance:testNGListeners1.TestCase1@7fee8714]" started-at="2018-09-01T19:05:31Z" name="AmazonTitleMatch" finished-at="2018-09-01T19:05:31Z" duration-ms="18" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at testNGListeners1.TestCase1.AmazonTitleMatch(TestCase1.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AmazonTitleMatch -->
        <test-method signature="AmazonTitleVerify()[pri:0, instance:testNGListeners1.TestCase1@7fee8714]" started-at="2018-09-01T19:05:31Z" name="AmazonTitleVerify" finished-at="2018-09-01T19:05:33Z" duration-ms="1752" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AmazonTitleVerify -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testNGListeners1.TestCase1@7fee8714]" started-at="2018-09-01T19:05:34Z" name="tearDown" finished-at="2018-09-01T19:05:34Z" duration-ms="504" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testNGListeners1.TestCase2@4229bb3f]" started-at="2018-09-01T19:05:34Z" name="tearDown" finished-at="2018-09-01T19:05:37Z" duration-ms="2385" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testNGListeners1.TestCase1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
